native/packages/react-native-bpk-component-banner-alert/readme.md

```diff
@@ -89,12 +90,17 @@ export default class App extends Component {
 
 ## Props
 
-| Property           | PropType            | Required | Default Value |
-| ------------------ | ------------------- | -------- | ------------- |
-| message            | string              | yes      | -             |
-| type               | oneOf(ALERT_TYPES)  | yes      | -             |
-| actionButtonLabel  | string              | no       | null          |
-| children           | node                | no       | null          |
-| dismissable        | bool                | no       | false         |
-| expanded           | bool                | no       | false         |
-| onAction           | func                | no       | () => null    |
+| Property                  | PropType           | Required | Default Value |
+| ------------------------- | ------------------ | -------- | ------------- |
+| message                   | string             | yes      | -             |
+| type                      | oneOf(ALERT_TYPES) | yes      | -             |
+| animateOnEnter            | bool               | no       | false         |
+| animateOnLeave            | bool               | no       | false         |
+| children                  | node               | no       | null          |
+| dismissable               | bool               | no       | false         |
+| dismissButtonLabel         | String             | no       | null          |
+| expanded                  | bool               | no       | false         |
+| onDismiss                 | func               | no       | null          |
+| onToggleExpanded          | func               | no       | null          |
+| show                      | bool               | no       | true          |
+| toggleExpandedButtonLabel | String             | no       | null          |
```

Comment:
`Required` should be a boolean instead of `yes/no`. I know this isn't part of this PR specifically but we should probably take care of it anyway.

Reply:
Also `PropType` is `string` in one place and `String` in another, should just be `string`.